content/programming_model_overview.tex

```diff
@@ -144,7 +144,7 @@
       data object on another symmetric data object.
   \item \OPR{All-to-All}: All \acp{PE} participating in the routine exchange
       a fixed amount of contiguous or strided data with all other \acp{PE}
-      in the active set.
+      in the team.
```

Comment:
@davidozog @kwaters4 This line was also updated by the front matter section committee. I am going to drop this change from the draft as the front matter committee change is more consistent across this section. Please review and let me know if you have any concerns.

Reply:
I had to rewrite history a bit to make the merges go seamlessly. This is my version of the collectives branch that dropped the above change: https://github.com/openshmem-org/specification/compare/master...jdinan:specification:collectives_section-v1.6-RC1

Reply:
Thanks @jdinan - sorry for the conflict.